### DISCUSSION TITLE
how can I require the faker-js in node js?

### DISCUSSION THREAD

**Discussion by md-ataur** [faker-js/faker#1776] [Q&A] [answered]
how can I require the faker-js in node js? Import doesn't work

**Comment by ST-DDT** [comment, marked as answer]
Did you follow the steps listed in our [guide](https://fakerjs.dev/guide/)?

````sh
npm install @faker-js/faker
````

````ts
import { faker } from '@faker-js/faker';
````

If that doesn't help please share your package.json, the respecitive script and the error message your are getting.
